usaspending_api/database_scripts/matview_generator/mv_search_award_all_pre2008.json

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
Similar to the suggestion of class inheritance for models, might there be a way to save on lines of code for all the JSON templates? 
Maybe a base template that gets interpolated to inject model-specific names/identifiers?

Reply:
It is certainly possible. However I would rather not since the JSON -> SQL is already convoluted enough